scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -363,8 +370,9 @@
                                         ; This can only occur in testing when you're e.g., blowing away the database.
                                         ; It will go through :missing,:missing and then be deleted from the map.
                                         ; TODO: May be evidence of a bug where we process pod changes when we're starting up.
-                                        :pod/running (kill-pod-in-weird-state compute-cluster pod-name
-                                                                              nil k8s-actual-state-dict)
+                                        ; Currently occurs because kill's can race ahead of launches, we kill something that has
+                                        ; been added to datomic, but hasn't been submitted to k8s yet.
```

Comment:
Above we say:

> This can only occur in testing when you're e.g., blowing away the database.

That seems to be in conflict with this.

Reply:
Fair. 'only' isn't applicable anymore.